src/styles/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    position: relative;
+    width: 40px;
+    height: 40px;
+    left: 50px;
```

Comment:
The position property of the header__logo class is set to relative, but you're only using the left property from the position attributes. Unless you're also using top, right, or bottom, you might not need to set the position to relative.